paper/src/main/java/vg/civcraft/mc/civmodcore/commands/CommandManager.java

```diff
@@ -8,21 +8,23 @@
 import co.aikar.commands.CommandCompletions.CommandCompletionHandler;
 import co.aikar.commands.CommandContexts;
 import com.google.common.base.Strings;
+
```

Comment:
?

Reply:
Ah, no idea. I'll remove the newline.

Reply:
Editorconfig updated, this should reflect a better style more in line with what you asked for.